src/js/angular-datepicker.js

```diff
@@ -454,11 +454,11 @@
 
         $scope.setDatepickerDay = function setDatepickeDay(day) {
 
-          if ($scope.isSelectableDate($scope.monthNumber, $scope.year, day)) {
```

Comment:
Hi, why you removed this,  this piece of code is needed to disable specific days... :)